pkg/steps/pre_publish_images.go

```diff
@@ -0,0 +1,121 @@
+package steps
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+
+	imageapi "github.com/openshift/api/image/v1"
+	"github.com/openshift/ci-operator/pkg/api"
```

Comment:
Can you please separate local imports from others?

Reply:
Oops yes